### ISSUE TITLE
Play button and other controls in video player not labeled for screen reader

### ISSUE THREAD

**Issue by drva** [Simon-Initiative/oli-torus#4105]
From Responsible AI meeting with Kimberly Norris

**Describe the bug**
Kimberly said that the play button of the video player was not labeled in screen reader, it just said clickable button. Likewise for the controls. 

**Expected behavior**
I expect the desired behavior is that play button and other controls should be labeled correctly as the controls that they are for screen readers. 
Note: Kimberly said she gave links to some open source accessible players; doesn't know if this is possible to consider but it would be her recommendation for fixing this fastest.

(I think this is significantly important for accessibility).

**Comment by bitbldr** [comment]
@drva will provide links if necessary
